src/sources/nats.rs

```diff
@@ -23,7 +23,7 @@ enum BuildError {
 #[serde(deny_unknown_fields)]
 pub struct NatsSourceConfig {
     url: String,
-    name: String,
+    connection_name: String,
```

Comment:
Could we alias this as `name` to avoid breaking backwards compatibility? You can just add `#[serde(alias = "name")]`.